bill-of-materials.json

```diff
@@ -143,6 +143,15 @@
 			}
 		]
 	},
+	{
+		"project": "github.com/golang-jwt/jwt",
```

Comment:
etcd already uses jwt/v4, please do not introduce out of date version.
https://github.com/etcd-io/etcd/blob/9d8000f374f68de6e4c6af97016be54b592f33dd/bill-of-materials.json#L147

Reply:
Oops. It looks like I had an incorrect import in one of my test cases. Resolved and rolled back the BOM changes.